tests/README.md

```diff
@@ -0,0 +1,90 @@
+MetalK8s is an infrastructure product, that rely on ansible so framework in python is somewhat “obvious”.
+Pytest is quite famous right famous.
```

Comment:
I'll only look at the content for now, not language :wink:

Reply:
"right now". yeah bear with me